Userland/Libraries/LibWeb/UIEvents/WheelEvent.idl

```diff
@@ -0,0 +1,24 @@
+// https://www.w3.org/TR/uievents/#idl-wheelevent
+[Exposed=Window]
+interface WheelEvent : MouseEvent {
```

Comment:
since this inherits from MouseEvent, the IDL file should `import` the MouseEvent idl.